src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -59,6 +59,14 @@ export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
   const supportArticleURL =
     "https://support.artsy.net/hc/en-us/articles/360048946973"
 
+  const authenticityGuaranteeImageURL =
+    "http://files.artsy.net/authenticityguaranteeartwork.jpg"
+  const securePaymentImageURL =
+    "http://files.artsy.net/securepaymentartwork.jpg"
+  const moneyBackGuaranteeImageURL =
+    "http://files.artsy.net/moneybackguaranteeartwork.jpg"
+  const heroImageURL = "http://files.artsy.net/buyerGuaranteeHeroImage.jpg"
```

Comment:
You're going to want to pass these through the resized function and pull out optimized versions along with the srcSets.

Reply:
The designer requested these images be used because they are resized according to the specs.  Is there a reason to resize them again?

Reply:
According to the specs with the expectation that they'll be run through our resize function maybe? Currently they are at  ~1600x1200 @1x and non-optimized. Resizing them to exactly the 1x display dimensions and utilizing the returned `srcSet` ensures they are correct .